src/api/app/models/repository.rb

```diff
@@ -2,6 +2,7 @@ class Repository < ApplicationRecord
   belongs_to :project, foreign_key: :db_project_id, inverse_of: :repositories
 
   before_destroy :cleanup_before_destroy
+  before_save :set_remote_project_name_to_empty_string
```

Comment:
Is this really needed? Can't we directly created with the a non null `remote_project_name`, so we don't have to run this every time we save?